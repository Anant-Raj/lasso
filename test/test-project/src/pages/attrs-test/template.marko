<lasso-page name="page1" package-path="./browser.json"/>

<html>
    <head>
        <lasso-slot name="css-slot" inline-style-attrs="{'my-custom': true}"/>
    </head>
    <body>
        <lasso-slot name="js-slot" inline-script-attrs="{'my-custom': true}"/>
    </body>
</html>
