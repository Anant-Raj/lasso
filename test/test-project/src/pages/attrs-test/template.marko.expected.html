<html><head><style my-custom>body .inline {
    background-color: red;
}</style></head><body><script my-custom>console.log('hello-inline');</script></body></html>